app/controllers/admin/sessions/list.js

```diff
@@ -1,71 +1,101 @@
-      title        : 'Starts At'
-    },
-    {
-      propertyName : 'endsAt',
-      template     : 'components/ui-table/cell/cell-simple-date',
-      dateFormat   : 'MMMM DD, YYYY - hh:mm A',
-      title        : 'Ends At'
-    },
-    {
-      template         : 'components/ui-table/cell/cell-simple-buttons',
-      title            : 'Actions',
-      disableSorting   : true,
-      disableFiltering : true
-    }
-  ],
-  actions: {
-    deleteSession(session) {
-      this.set('isLoading', true);
-      session.destroyRecord()
-        .then(() => {
-          this.notify.success(this.l10n.t('Session has been deleted successfully.'));
-        })
-        .catch(() => {
-          this.notify.error(this.l10n.t('An unexpected error has occurred.'));
-        })
-        .finally(() => {
-          this.set('isLoading', false);
-        });
-    },
-    editSession(session_id, event_id) {
-      this.transitionToRoute('events.view.sessions.edit', event_id, session_id);
-    },
-    viewSession(id) {
-      this.transitionToRoute('my-sessions.view', id);
+import { computed, action } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+  @computed()
+  get columns() {
+    return [
+      {
+        name      : 'Event Name',
+        valuePath : 'name'
+      },
+      {
+        name      : 'Title',
+        valuePath : 'title'
+      },
+      {
+        name          : 'State',
+        valuePath     : 'status',
+        cellComponent : 'ui-table/cell/events/view/sessions/cell-session-state'
+      },
+      {
+        name          : 'Speakers',
+        valuePath     : 'speakers',
+        cellComponent : 'ui-table/cell/cell-speakers'
+      },
+      {
+        name          : 'Submitted At',
+        valuePath     : 'submittedAt',
+        cellComponent : 'ui-table/cell/cell-simple-date',
+        options       : {
+          dateFormat: 'MMMM DD, YYYY - hh:mm A'
+        }
+      },
+      {
+        name          : 'Starts At',
+        valuePath     : 'startsAt',
+        cellComponent : 'ui-table/cell/cell-simple-date',
+        options       : {
+          dateFormat: 'MMMM DD, YYYY - hh:mm A'
+        }
+      },
+      {
+        name          : 'Ends At',
+        valuePath     : 'endsAt',
+        cellComponent : 'ui-table/cell/cell-simple-date',
+        options       : {
+          dateFormat: 'MMMM DD, YYYY - hh:mm A'
+        }
+      },
+      {
+        name          : 'Actions',
+        cellComponent : 'ui-table/cell/cell-simple-buttons',
+        valuePath     : 'actions',
+        actions       : {
+          deleteSession : this.deleteSession.bind(this),
+          editSession   : this.editSession.bind(this),
+          viewSession   : this.viewSession.bind(this)
+        }
+      }
+    ];
+  }
+
+  @computed('model.[]', 'model.@each.event')
+  get rows() {
```

Comment:
Use model directly. One less computed property. One less iteration of the dataset.